Twig/Extension/SonataDoctrinePHPCRAdminExtension.php

```diff
@@ -17,55 +17,13 @@
 class SonataDoctrinePHPCRAdminExtension extends \Twig_Extension
 {
     /**
-     * @var \Twig_Environment
-     */
-    protected $environment;
-
-    /**
-     * {@inheritdoc}
-     */
-    public function initRuntime(\Twig_Environment $environment)
-    {
-        $this->environment = $environment;
-    }
-
-    /**
-     * {@inheritdoc}
-     */
-    public function getName()
-    {
-        return 'sonata_doctrine_phpcr_admin';
-    }
-
-    /**
-     * render a list element from the FieldDescription.
-     *
-     * @param object                    $object
-     * @param FieldDescriptionInterface $fieldDescription
-     * @param array                     $params
-     *
-     * @return string
-     */
-    public function renderListElement($object, FieldDescriptionInterface $fieldDescription, $params = array())
```

Comment:
this was added in https://github.com/sonata-project/SonataDoctrinePhpcrAdminBundle/commit/638ee10b84976650a7b35777c024dcd185aeb742 but it isn't called from anywhere. To me, this seems like dead code that should be removed.